modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/pages/hearing_type_and_additional_issues.rb

```diff
@@ -22,7 +22,7 @@ def build!
             return pdf unless extra_issues?
 
             pdf.text("\n<b>Additional Issues</b>\n", inline_format: true)
-            pdf.table(extra_issues_table_data, header: true)
+            pdf.table(extra_issues_table_data, width: 540, header: true)
```

Comment:
What is the value 540 based off of?

Reply:
540 is the number of pixels that created an even margin on either side of the table, and left it just smaller than the margins of the table/columns of text above.